hack/W2L/jenkins-w2l-script.sh

```diff
@@ -0,0 +1,207 @@
+#!/bin/sh
+
+# Jenkins CI script for Windows to Linux CI.
+# Heavily modified by John Howard (@jhowardmsft) December 2015 to try to make it more reliable.
+SCRIPT_VER="7-Dec-2015 18:08"
+
+
+ec=0
+echo INFO: Started at `date`. Script version $SCRIPT_VER
+
+# get the ip
+ip="${DOCKER_HOST#*://}"
+ip="${ip%%:*}"
+
+# make sure it is the right DOCKER_HOST
+export DOCKER_HOST="tcp://$ip:2357"
```

Comment:
Just curious; why is this using a non-standard port (I.e. 2357 in stead of 2375)? Just curious, as it could be either of a reason, or a typo :smile: 

(If it's on purpose, perhaps this should have a comment explaining)